pkg/csi/service/vanilla/controller_helper.go

```diff
@@ -231,3 +231,55 @@ func getVolumeManagerForVolumeID(ctx context.Context, controller *controller, vo
 	}
 	return volumeManager, nil
 }
+
+// getVCenterManagerForVolumeID returns vCenter manager for the given volumeId.
+// If multi-vcenter-csi-topology feature is disabled, legacy vCenter manager is returned
+// from `controller.manager.VCenterManager`.
+// If multi-vcenter-csi-topology feature is enabled, vCenter manager is returned from
+// `controller.managers.VCenterManager``.
+func getVCenterManagerForVolumeID(ctx context.Context, controller *controller, volumeId string,
+	volumeInfoService cnsvolumeinfo.VolumeInfoService) (vsphere.VirtualCenterManager, error) {
+	var vCenterManager vsphere.VirtualCenterManager
+	if multivCenterCSITopologyEnabled {
+		vCenterManager = controller.managers.VcenterManager
```

Comment:
We will need to pick the right VC manager from the map controller.managers.VcenterConfigs[<VC host>]. We're not using volumeID at all in this function right now.

Reply:
when multi vCenter is enabled, get vCenter manager directly using `c.managers.VcenterManager` else use `c.manager.VcenterManager`
We do not need this function.

Reply:
> when multi vCenter is enabled, get vCenter manager directly using `c.managers.VcenterManager` else use `c.manager.VcenterManager` We do not need this function.

Added this function to avoid multiple if-else in code

Reply:
> We will need to pick the right VC manager from the map controller.managers.VcenterConfigs[]. We're not using volumeID at all in this function right now.

will remove unnecessary arguments and return values